Assets/Scripts/Cheat/Main/TowerController.cs

```diff
@@ -0,0 +1,125 @@
+﻿/*
+ * Author(s): Isaiah Mann
+ * Description: Controls all the towers in the game
+ */
+
+using UnityEngine;
+using SimpleJSON;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.EventSystems;
+
+public class TowerController : UnitController<ITower, Tower, TowerList>, ITowerController {
+	public const string TOWER_TAG = "Tower";
+	const float DRAG_HEIGHT_OFFSET = 1.25f;
+
+	public GameObject CoreOrbPrefab;
+	public GameObject CoreOrbInstance;
+	GameObject potentialPurchaseTower = null;
+	MapTileBehaviour previousHighlightedMapTile = null;
+	HashSet<TowerBehaviour> activeTowers = new HashSet<TowerBehaviour>();
+	public static List<TowerBehaviour> healTowers = new List<TowerBehaviour> (); //9/29/16
+
```

Comment:
Slightly unsure why it marks this whole file as changed. Did you manually upload the file?